src/CommunityToolkit.Maui/AppThemeResource.shared.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// Represents a resource that is aware of the operating system theme.
+/// </summary>
+public class AppThemeResource : AppThemeObject<object>
```

Comment:
Can we make this `sealed` for the same reason as the `AppThemeColor`?

```suggestion
public sealed class AppThemeResource : AppThemeObject<object>
```